dispatcher/backend/src/routes/requested_tasks/requested_task.py

```diff
@@ -73,6 +73,9 @@ def list_of_requested_tasks(session: so.Session, token: AccessToken.Payload = No
     skip, limit = request_args["skip"], request_args["limit"]
     schedule_names = request_args["schedule_name"]
     priority = request_args.get("priority")
+    # Get sorting parameters
+    sort_by = request_args.get("sort_by", "priority")
```

Comment:
Why do we have defaults? Simple default like no sort_by and no sort_order seems way more semantically correct / simple to read.